pyiceberg/catalog/sql.py

```diff
@@ -110,8 +114,12 @@ def __init__(self, name: str, **properties: str):
 
         if not (uri_prop := self.properties.get("uri")):
             raise NoSuchPropertyException("SQL connection URI is required")
-        echo = bool(self.properties.get("echo", False))
-        self.engine = create_engine(uri_prop, echo=echo)
+
+        echo_str = str(self.properties.get("echo", DEFAULT_ECHO_VALUE)).lower()
+        echo = strtobool(echo_str) if echo_str != "debug" else "debug"
+        pool_pre_ping = strtobool(self.properties.get("pool_pre_ping", DEFAULT_PRE_PING_VALUE))
```

Comment:
sorry didnt catch this the last review. 
very much nit, `DEFAULT_PRE_PING_VALUE` -> `DEFAULT_POOL_PRE_PING_VALUE` to follow the convention `DEFAULT_{variable_name}_VALUE`

Reply:
nitpicking is good, no worries.